pkg/datasource/columns.go

```diff
@@ -197,3 +205,71 @@ func (ds *dataSource) Columns() (*columns.Columns[DataTuple], error) {
 	}
 	return cols, nil
 }
+
+var defaultFieldAnnotations = map[string]string{
+	ColumnsWidthAnnotation:     "16",
+	ColumnsEllipsisAnnotation:  string(metadatav1.EllipsisEnd),
+	ColumnsAlignmentAnnotation: string(metadatav1.AlignmentLeft),
+}
+
+var annotationsTemplates = map[string]map[string]string{
+	"timestamp": {
+		ColumnsWidthAnnotation:    "35",
+		ColumnsMaxWidthAnnotation: "35",
+		ColumnsEllipsisAnnotation: "end",
+	},
+	"node": {
+		ColumnsWidthAnnotation:    "30",
+		ColumnsEllipsisAnnotation: string(metadatav1.EllipsisMiddle),
+	},
+	"pod": {
+		ColumnsWidthAnnotation:    "30",
+		ColumnsEllipsisAnnotation: string(metadatav1.EllipsisMiddle),
+	},
+	"container": {
+		ColumnsWidthAnnotation: "30",
+	},
+	"namespace": {
+		ColumnsWidthAnnotation: "30",
+	},
+	"containerImageName": {
+		ColumnsWidthAnnotation: "30",
+	},
+	"containerImageDigest": {
+		ColumnsWidthAnnotation: "30",
+	},
+	"containerStartedAt": {
+		ColumnsHiddenAnnotation: "true",
+		ColumnsWidthAnnotation:  "35",
+	},
+	"comm": {
+		DescriptionAnnotation:     "Process name",
+		ColumnsMaxWidthAnnotation: "16",
+	},
+	"pid": {
+		ColumnsMinWidthAnnotation:  "7",
+		ColumnsAlignmentAnnotation: string(metadatav1.AlignmentRight),
```

Comment:
nit: Why alignment right for these fields?

Reply:
I find it easier to read integer values when they are aligned to the right.

Reply:
Ok.